articles/app-service/app-service-key-vault-references.md

```diff
@@ -41,7 +41,7 @@ A Key Vault reference is of the form `@Microsoft.KeyVault({referenceString})`, w
 > | VaultName=_vaultName_;SecretName=_secretName_;SecretVersion=_secretVersion_ | The **VaultName** should the name of your Key Vault resource. The **SecretName** should be the name of the target secret. The **SecretVersion** should be the version of the secret to use. |
 
 > [!NOTE] 
-> Versions are currently required. When rotating secrets, you will need to update the version in your application configuration.
+> Versions are currently not required. If you are rotating secrets you should not include the version ( i.e. @Microsoft.KeyVault(SecretUri=https://myvault.vault.azure.net/secrets/mysecret)
```

Comment:
The note on versions today is as desired. I'd ask that these changes be reverted.